backend/application/syson-application/src/test/java/org/eclipse/syson/application/export/checker/SysmlImportExportChecker.java

```diff
@@ -45,8 +48,13 @@ public class SysmlImportExportChecker {
 
     private final SysMLv2DocumentExporter exporter;
 
-    public SysmlImportExportChecker(SysMLExternalResourceLoaderService sysmlLoader, IEditingDomainFactory editingDomainFactory, SysMLv2DocumentExporter exporter) {
+    private final SysMLEditingContextProcessor sysMLEditingContextProcessor;
+
```

Comment:
Why do you add 3 empty lines? Please remove them and let only one.

Reply:
Done.